### ISSUE TITLE
Problems searching for annotations across packages

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NickstaDB/reflections#86]
```
What steps will reproduce the problem?
1. A class is annotated with an annotation outside its immediate package.
2. Configure reflections to search the immediate package.

What is the expected output? What do you see instead?
Expected: A Set containing the annotated class(es).
Actual: An empty set.

What version of the product are you using? On what operating system?
0.9.5-RC2

Please provide any additional information below.

See the test case here: https://github.com/ProjectMoon/reflections-bug

In this particular test case, searching the "parent" package (where the 
annotation happens to be located) will find my annotated classes. That might 
not be the specific problem though. It may be a larger problem, e.g. I did not 
test searching across completely unrelated packages (like let's say annotation 
is in com.site.pkg and it's being used in .net.othersite.something).

```

Original issue reported on code.google.com by `kni...@gmail.com` on 4 Sep 2011 at 11:08

**Comment by GoogleCodeExporter** [comment]
```
Wondering if there's anything going on with this issue?
```

Original comment by `kni...@gmail.com` on 21 Sep 2011 at 8:09

**Comment by GoogleCodeExporter** [comment]
```
Hi.. Just want to inform that this is works for me:
https://gist.github.com/1824474

Sorry, just re-use your github repo because I'm not familiar with ivy.
```

Original comment by `xsalefter@gmail.com` on 14 Feb 2012 at 7:28

**Comment by GoogleCodeExporter** [comment]
```
What version did you test the fix with?
```

Original comment by `kni...@gmail.com` on 14 Feb 2012 at 1:53

**Comment by GoogleCodeExporter** [comment]
```
fixed on 0.9.7.RC1 and trunk. comments?
```

Original comment by `ronm...@gmail.com` on 2 May 2012 at 3:56
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed 0.9.7
```

Original comment by `ronm...@gmail.com` on 22 May 2012 at 7:47
- Changed state: **Fixed**
